connector/MongoDBAtlasODBC.pq

```diff
@@ -15,12 +15,22 @@ MongoDBAtlasODBCType = type function(
         Documentation.FieldDescription = "Database to connect to",
         Documentation.SampleValues = {"db"}
     ]),
+    optional query as (type text meta [
+        Documentation.FieldCaption = "SQL Statement",
+        Documentation.SampleValues = {"SELECT * from table"},
+        Formatting.IsMultiLine = true,

```

Comment:
Do you know if there is a size limit for the text field?

Reply:
I could not find a size limit for the input text field.  I'll add that to our test plan as something to test.
The docs do point out the maximum length of the text data type in terms of loaded data as 268,435,456 Unicode characters ([link](https://learn.microsoft.com/en-us/power-query/data-types)) [(link](https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-data-types#text-type))